### PR TITLE
fix(components-angular): Add hidden text to intranet header logo

### PR DESCRIPTION
**Pull request by b1aserlu**
adds a hidden text to the postlogo of the intranet header

### CODE REVIEW

File: .changeset/wet-lions-fly.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-intranet-header': patch
+---
+
+Added text to the postlogo for accessibility
```
Comment (gfellerph) [inline .changeset/wet-lions-fly.md:5]:
```suggestion
Added a descriptive text to the Post logo home link
```

**Review by gfellerph — DISMISSED** [review]
Just a small nit for the changeset. Looks good to me otherwise.

Generally, it's not a good idea to use function calls in Angular templates (see https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496). The header will be replaced as soon as possible, therefore I don't think it's worth it to refactor here.

**Comment by gfellerph** [comment]
@alizedebray, we got requests for other languages in the intranet header (like Portuguese). What's the best way to handle the translation in an extensible way? Just request the text as a prop?

File: .changeset/wet-lions-fly.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-intranet-header': patch
+---
+
+Added a descriptive text to the Post logo home link
```
Comment (alizedebray) [inline .changeset/wet-lions-fly.md:5]:
```suggestion
Added a descriptive text to the Post logo home link.
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts [hunk 3]
```diff
@@ -65,6 +66,12 @@ export class SwissPostIntranetHeaderComponent implements OnInit, OnChanges, Afte
       it: 'Cercare in intranet',
       en: 'Browse the intranet',
     },
+    postlogo: {
```
Comment (alizedebray) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts:69]:
```suggestion
    postLogo: {
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts [hunk 4]
```diff
@@ -317,6 +324,10 @@ export class SwissPostIntranetHeaderComponent implements OnInit, OnChanges, Afte
     }
   }
 
+  public getPostlogoText() {
```
Comment (alizedebray) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts:327]:
```suggestion
  public getPostLogoText() {
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts [hunk 5]
```diff
@@ -317,6 +324,10 @@ export class SwissPostIntranetHeaderComponent implements OnInit, OnChanges, Afte
...
+    return this.localization['postlogo'][this.lang.toLocaleLowerCase()];
```
Comment (alizedebray) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts:328]:
```suggestion
    return this.localization['postLogo'][this.lang.toLocaleLowerCase()];
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html [hunk 6]
```diff
@@ -8,7 +8,9 @@
       <div class="intranet-header">
         <div class="d-flex align-items-center flex-nowrap">
           <div class="bg-logo d-sm-block">
-            <a [href]="this.logoUrl" class="navbar-brand header-logo"></a>
+            <a [href]="this.logoUrl" class="navbar-brand header-logo">
+              <span class="visually-hidden">{{getPostlogoText()}}</span>
```
Comment (alizedebray) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html:12]:
```suggestion
              <span class="visually-hidden">{{getPostLogoText()}}</span>
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts [hunk 7]
```diff
@@ -52,6 +52,7 @@ export class SwissPostIntranetHeaderComponent implements OnInit, OnChanges, Afte
   localization: {
     moreLabel: { [key: string]: string };
     searchPlaceholder: { [key: string]: string };
+    postlogo: { [key: string]: string };
```
Comment (b1aserlu) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts:55]:
```suggestion
    postLogo: { [key: string]: string };
```

**Review by alizedebray — APPROVED** [review]

**Comment by alizedebray** [comment]
> @alizedebray, we got requests for other languages in the intranet header (like Portuguese). What's the best way to handle the translation in an extensible way? Just request the text as a prop?

@gfellerph a prop through which you can specify a path to a JSON file seems to be a good solution that is not too tied to Angular so that we can implement the same logic when we move the header to a web component.

**Pull request merged**